pkg/k8s/service_cache.go

```diff
@@ -129,26 +130,26 @@ func (s *ServiceCache) GetServiceFrontendIP(svcID ServiceID, svcType loadbalance
 	s.mutex.RLock()
 	defer s.mutex.RUnlock()
 	svc := s.services[svcID]
-	if svc == nil || svc.Type != svcType {
+	if svc == nil || svc.Type != svcType || len(svc.FrontendIPs) == 0 {
 		return nil
 	}
 
-	return svc.FrontendIP
+	return svc.FrontendIPs[0]
```

Comment:
same here: Why are we picking the first element of `FrontendIPs`?

Reply:
Same as above.

Reply:
Same answer before